File: AC/ABC-TDD/180/c.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var n, tmp int
	var ans []int
	fmt.Scan(&n)
	for i := 1; i <= n; i++ {
		if i*i > n {
			break
		}
		if n%i == 0 {
			ans = append(ans, i)
			tmp = n / i
			if tmp != i {
				ans = append(ans, tmp)
			}
		}
	}
	sort.Ints(ans)
	for _, nums := range ans {
		fmt.Println(nums)
	}
}

File: AC/ABC-TDD/180/c.py
n = int(input())
ans = []
for i in range(1, n+1):
    if i*i > n:
        break
    if n % i == 0:
        ans.append(i)
        tmp = n//i
        if i != tmp:
            ans.append(n//i)
ans = sorted(ans)
counts = len(ans)
for num in ans:
    print(num)
